README.md

```diff
@@ -4579,27 +4579,33 @@ try controller.performFetch()
 
 **GRDB can encrypt your database with [SQLCipher](http://sqlcipher.net) v3.4.0.**
 
-This requires a manual installation of GRDB:
+Using CocoaPods
+```ruby
+pod 'GRDB.swift/SQLCipher'
+```
+
+Using a manual installation of GRDB:
 
 1. Clone the GRDB.swift git repository, checkout the latest tagged version, and download SQLCipher sources:
     
     ```sh
     cd [GRDB.swift directory]
     git checkout v0.107.0
     git submodule update --init SQLCipher/src
-    ````
+    ```
     
 2. Embed the `GRDB.xcodeproj` project in your own project.
 
 3. Add the `GRDBCipherOSX` or `GRDBCipheriOS` target in the **Target Dependencies** section of the **Build Phases** tab of your application target.
 
 4. Add the `GRDBCipher.framework` from the targetted platform to the **Embedded Binaries** section of the **General**  tab of your target.
 
+5. Import `GRDBCipher` module in your project's files
```

Comment:
"That step is in the manual installation steps", because you moved it there.

Currently GRDB exposes three different modules: GRDB, GRDBCipher, and GRDBCustom (for [custom SQLite builds](https://github.com/groue/GRDB.swift/blob/master/Documentation/CustomSQLiteBuilds.md)). I'd rather be *consistent*: either expose a single module name `GRDB` for all frameworks and installation methods (but this may create a new set of problems), or use the `GRDBCipher` module name for GRDB+SQCipher whatever the installation method.

So yes, please, if CocoaPods supports it, I'd like a specific `module_name`. And you will be able to restore the `import GRDBCipher` line in its previous place in the README.